src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/get_interior_vehicle_data_request_test.cc

```diff
@@ -590,4 +590,123 @@ TEST_F(GetInteriorVehicleDataRequestTest,
+  const std::uint32_t chanel1_index = 1u;
+  const std::uint32_t chanel2_index = 2u;
+  const std::uint32_t chanel3_index = 3u;
+
+  // Arrange
+  MessageSharedPtr mobile_message = CreateBasicMessage();
+
+  MessageSharedPtr hmi_response_message = CreateBasicMessage();
+  auto& hmi_response_params = (*hmi_response_message)[strings::msg_params];
+  hmi_response_params[hmi_response::code] = hmi_apis::Common_Result::SUCCESS;
+  hmi_response_params[strings::connection_key] = kAppId;
+
+  auto& msg_params = (*hmi_response_message)[strings::msg_params];
+  msg_params[message_params::kModuleType] = module_type;
+
+  smart_objects::SmartObject availableHDchanels =
+      smart_objects::SmartObject(smart_objects::SmartType_Array);
+
+  availableHDchanels[0] = chanel1_index;
+  availableHDchanels[1] = chanel2_index;
+  availableHDchanels[2] = chanel3_index;
+
+  msg_params[message_params::kModuleData][message_params::kRadioControlData]
+            [message_params::kAvailableHdChannels] = availableHDchanels;
+
+  ON_CALL(mock_interior_data_cache_, Contains(_)).WillByDefault(Return(false));
+  ON_CALL(mock_interior_data_manager_, CheckRequestsToHMIFrequency(_))
+      .WillByDefault(Return(true));
+
+  auto message_to_mob = CreateBasicMessage();
+
+  // Expectations
+  EXPECT_CALL(mock_rpc_service_, ManageHMICommand(_)).WillOnce(Return(true));
+  EXPECT_CALL(mock_rpc_service_, ManageMobileCommand(_, _))
+      .WillOnce(DoAll(SaveArg<0>(&message_to_mob), Return(true)));
+
+  // Act
+  auto command = CreateRCCommand<GetInteriorVehicleDataRequest>(mobile_message);
+  application_manager::event_engine::Event event(
+      hmi_apis::FunctionID::RC_GetInteriorVehicleData);
+
+  command->Init();
+  command->Run();
+
+  event.set_smart_object(*hmi_response_message);
+  command->on_event(event);
+
+  auto& HDchanels = (*message_to_mob)
+      [strings::msg_params][message_params::kModuleData]
+      [message_params::kRadioControlData][message_params::kAvailableHdChannels];
+  const size_t array_length = HDchanels.length();
+
+  EXPECT_EQ(array_length, 3u);
+
+  EXPECT_EQ(chanel1_index, HDchanels[0].asUInt());
+  EXPECT_EQ(chanel2_index, HDchanels[1].asUInt());
+  EXPECT_EQ(chanel3_index, HDchanels[2].asUInt());
+}
+
+TEST_F(GetInteriorVehicleDataRequestTest,
+       OnEvent_ValidHmiResponse_ClimateEnableAvailable) {
+  using rc_rpc_plugin::commands::GetInteriorVehicleDataRequest;
+  namespace hmi_response = application_manager::hmi_response;
+  namespace strings = application_manager::strings;
+
+  // Arrange
+  MessageSharedPtr mobile_message = CreateBasicMessage();
+
+  MessageSharedPtr hmi_response_message = CreateBasicMessage();
+  auto& hmi_response_params = (*hmi_response_message)[strings::msg_params];
+  hmi_response_params[hmi_response::code] = hmi_apis::Common_Result::SUCCESS;
+  hmi_response_params[strings::connection_key] = kAppId;
+
+  auto& msg_params = (*hmi_response_message)[strings::msg_params];
+
+  smart_objects::SmartObject climateControlData =
```

Comment:
@IGapchuk 
- named wrongly according to SDL Coding Style

Reply:
@AByzhynar Fixed in a57b72856e60e30b65c6435d169cee8d10729660.